index.css

```diff
@@ -0,0 +1,330 @@
+.background {
+  background-image: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  width: 1440px;
```

Comment:
You do not need to set an explicit width like you did here, please follow up with other places were u set explicit width and make sure they are not affecting the page display